lms/djangoapps/instructor_task/api.py

```diff
@@ -46,13 +49,34 @@
 from util import milestones_helpers
 
 
+TASK_TYPE_EXPORT_GRADES_TO_RGB = "export_grades_to_rgb"
+
+
 class SpecificStudentIdMissingError(Exception):
     """
     Exception indicating that a student id was not provided when generating a certificate for a specific student.
     """
     pass
 
 
+def get_running_instructor_rgb_tasks(course_id, user=None):
+    """
+    Returns a query of InstructorTask objects of running tasks for remote grade book.
+
+    Used to generate a list of tasks to display on the instructor dashboard.
+    """
+    # list down remote grade book tasks
+    now = datetime.datetime.now()
+    return InstructorTask.objects.filter(
+        course_id=course_id,
+        task_state__icontains="success",
+        task_type=TASK_TYPE_EXPORT_GRADES_TO_RGB,
+        updated__lte=now,
+        updated__gte=now - datetime.timedelta(days=5),
+        requester=user
+    ).order_by('-updated')[0:3]
```

Comment:
We still want to see other running tasks in the remote gradebook page, right? For example, if a CSV export was currently in progress, we would want to see that. We should first call `get_running_instructor_tasks` and then add these grade export success tasks to that list (and remove duplicates)

Also, we should make `user` a required argument (no default `None`)